account_payment_other_company/models/account_payment.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountPayment(models.Model):
+    _inherit = 'account.payment'
+
+    other_journal_id = fields.Many2one('account.journal',
+                                       string='Paid By')
```

Comment:
@osi-scampbell We need a computed domain to get all the bank and cash journals from any company.

Something like:
```
domain=lambda self: self.sudo()._compute_domain_other_journal()

def _compute_domain_other_journal(self):
    return [('type', 'in', ('bank', 'cash'))]
```

Reply:
@max3903 Ok I think I understand, we need to take the domain off of the view and move it to the model itself, that way we can call sudo(). That will take care of the your previous comment above correct?